HACKING.md

```diff
@@ -35,67 +35,72 @@ Enter the toolbox with
 
     toolbox enter cockpit
 
-Your home directory, user D-Bus, etc. are shared with the host, so you can e.g.
-edit files as you usually do. Building and running tests happens inside the
-toolbox. If desired, you can install additional packages with `sudo dnf install`.
+Your home directory, user D-Bus, etc. are shared with the host, so you can
+edit files as you normally would. Building and running tests happens inside the
+toolbox container. If desired, you can install additional packages with
+`sudo dnf install`.
 
 ## Working on Cockpit's session pages
 
-Most contributors want to work on the web (HTML/JavaScript) parts of Cockpit.
-First install Cockpit on your local machine as described in:
+Most contributors want to work on the web (HTML, JavaScript, CSS) parts of Cockpit.
+First, install Cockpit on your local machine as described in:
 
 <https://cockpit-project.org/running.html>
 
 Cockpit uses a lot of build and runtime dependencies from [npm](https://www.npmjs.com). Get them with
 
     tools/node-modules checkout
 
-Next run this command from your top level Cockpit checkout directory, and make
+Next, run this command from your top level Cockpit checkout directory, and make
 sure to run it as the same user that you'll use to log into Cockpit below.
 
     mkdir -p ~/.local/share/
     ln -s $(pwd)/dist ~/.local/share/cockpit
 
-This will cause cockpit to read JavaScript and HTML files directly from the
-built package output directory instead of using the installed Cockpit UI files.
+This will cause cockpit to read JavaScript, HTML, and CSS files directly from the
+locally built package output directory instead of using the system-installed Cockpit
+files.
 
 Now you can log into Cockpit on your local Linux machine at the following
 address. Use the same user and password that you used to log into your Linux
 desktop.
 
 <http://localhost:9090>
 
-After every change to your sources the bundles need to be rebuilt. The
-recommended and fastest way is to do that in "watch" mode on the particular
-page that you are working on, see [pkg/](./pkg/) for which pages exist.
-E.g. if you want to work on anything in [pkg/systemd](./pkg/systemd/), run:
+After every change to the source files, bundles need to be rebuilt. The
+recommended and fastest way is to do that is using the "watch" mode on 
+the page that you are working on. For example, if you want to work on 
+anything in [pkg/systemd](./pkg/systemd/), run:
 
     tools/webpack-watch systemd
 
-Note that this enables eslint by default -- if you want to disable it, run it
-with `-e`/`--no-eslint`.
+See [pkg/](https://github.com/cockpit-project/cockpit/tree/main/pkg)
```

Comment:
Oh, I guess I missed this:

```suggestion
See [pkg/](./pkg/)
```

Reply:
I changed this and force-pushed after rebasing, but I don't know why GitHub isn't showing my changes — both here and as a force-push.

Reply:
Aha! It just took _several minutes_. (Usually, it's pretty instant.)